posts/templates/posts/_post.html

```diff
@@ -37,23 +39,38 @@
     <div class="actions">
         {% if request.user.is_authenticated %}
             <a class="mdi-action-grade{% if post.extra.is_favorited %} favorite{% endif %}"
-               href="{% url 'posts:favorite' pk=post.original_or_self.pk %}"></a>
+               href="{% url 'posts:favorite' pk=post.original_or_self.pk %}">
+            </a>
             {% if post.extra.can_be_reposted %}
-                <a class="mdi-av-repeat"
-                   href="{% url 'posts:repost' pk=post.original_or_self.pk %}"></a>
+                <a class="mdi-av-repeat" href="{% url 'posts:repost' pk=post.original_or_self.pk %}"></a>
             {% endif %}
+            <a class="mdi-content-reply" href="#reply-area-{{ post.pk }}" data-toggle="collapse"></a>
             <div class="pull-right">
                 {% if post.extra.can_be_edited %}
-                    <a class="mdi-editor-mode-edit"
-                       href="{% url 'posts:edit' pk=post.original_or_self.pk %}"></a>
+                    <a class="mdi-editor-mode-edit" href="{% url 'posts:edit' pk=post.original_or_self.pk %}"></a>
                 {% endif %}
                 {% if post.extra.can_be_deleted %}
-                    <a class="mdi-action-delete"
-                       href="{% url 'posts:delete' pk=post.original_or_self.pk %}"></a>
+                    <a class="mdi-action-delete" href="{% url 'posts:delete' pk=post.original_or_self.pk %}"></a>
+                {% endif %}
+                {% if post.extra.can_be_replied %}
+                    <a class="" href="{% url 'posts:reply' pk=post.original_or_self.pk%}"></a>
```

Comment:
This should be in the left options together with favorite and repost, not with the right options like edit and delete.